node/core/proposer/src/lib.rs

```diff
@@ -209,9 +212,14 @@ where
 
 			drop(_span);
 
+			let inclusion_inherent_data = (
+				self.parent_header,
```

Comment:
If we put this at the end, it'll be easier to restart Rococo.

As is, I'm not sure how we should plan to upgrade Rococo. Do you think we should just restart it? Do you have more breaking changes planned?

Whether we upgrade the node-side or the runtime first, the `inclusion_inherent::create_inherent` will panic when it fails to decode the inclusion inherent.

Reply:
That's the last one (I hope :) ). I have some stuff in pipeline but it's not critical and we can get away with runtime only upgrade.